_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,135 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each `<meta name="viewport">` element with a `content` attribute.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- either does not have the property `user-scalable` or has it set to yes; and
+- does not have the property `maximum-scale`. 
+
+## Assumptions
+
+_There are currently no assumptions._
```

Comment:
There is, you're making an assumption that there is content in the page that there is content in the page for which two-dimensional scrolling doesn't provide meaning. Or in the case of the reflow SC, that there is text in the page. You can either put that in the assumption, or you can address it in the applicability, either way works for me.

Reply:
Adding to Wilco's point, you might make part of the applicability that there is [visible](#visible) content on the page. (don't know if "content" then also needs a definition...)